system-addon/lib/TopStoriesFeed.jsm

```diff
@@ -151,6 +151,15 @@ this.TopStoriesFeed = class TopStoriesFeed {
       case at.UNINIT:
         this.uninit();
         break;
+      case at.SET_PREF:
```

Comment:
This isn't necessary, although you need to add a `FEED_INIT` case (look at the dummy feed to see how to do it)